PythonRegression/tests/features/steps/api_test_steps.py

```diff
@@ -10,14 +12,42 @@
 config = {}
 responses = {'getNodeInfo':{},'getNeighbors':{},'getTips':{},'getTrytes':{}}   
 
+yamlPaths = ["./tests/features/machine1/config.yml","./tests/features/machine2/config.yml"]
+
+#Configuration
+@before.all
+def configuration():
+    machines = {}    
+    for i in range(len(yamlPaths)):
+        stream = open(yamlPaths[i],'r')
+        machine = load(stream,Loader=Loader)
+        world.seeds = machine.get('seeds')
+    
+        nodes = {}
+        keys = machine.keys()  
+        count = 1     
+        for i in keys:
+            if i != 'seeds' and i != 'defaults':
+                name = i
+                host = machine[i]['host']
+                nodes[name] = host
+
+        machine_tag = 'machine'+str(count)
+        machines[machine_tag] = nodes
+        count += 1
+          
+    world.machines = machines
+
+
```

Comment:
The configuration step for the files just stores the host and ports for each node in each machine in a dictionary for access by the tests at later points. The `@before_all` configuration step stores them all before each run of aloe. If you run each feature file separately then you can forego storing the machines, but if you intend to run multiple tests with one aloe command across multiple machines, it will only run the configuration the one time.